Src/BootCamp.Chapter1/Program.cs

```diff
@@ -1,9 +1,27 @@
-﻿namespace BootCamp.Chapter1
+﻿using System;
+
+namespace BootCamp.Chapter1
 {
     class Program
     {
         static void Main(string[] args)
         {
+           // int[] myArray = new int[] { 4, 8, 0, 1, 9, 3, 7, 2, 88, 54, 11, 18, 100, 6 };
+              int[] myArray = new int[] { 0, 1, 2, 3, 4, 5, 6, 7, 8, 9  };
+            //{ 1, 2, 3, 4, 5, 6, 7, 8, 9, 10 };
+            int[] testArray = new int[] { 0, 1 };
+           // int[] nullArray = new int[] { };
+
+
+            //ArrayOperations.Sort(myArray);
+            ArrayOperations.Reverse(myArray);
+            //ArrayOperations.RemoveLast(myArray);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this commented out code.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4905609)